### PR TITLE
CCP4I2 to use contact file and fix of failing unitttest

### PR DESCRIPTION
**Pull request by linucks**

### CODE REVIEW

File: i2/AMPLE.def.xml [hunk 1]
```diff
@@ -136,10 +136,24 @@
...
+			  	<qualifiers>
+			    	<title>Contact Format</title>
+			    	<guiLabel>The format of the contact file</guiLabel>
+			    	<onlyEnumerators>True</onlyEnumerators>
+			    	<menuText>one.two</menuText>
+			    	<enumerators>one,two</enumerators>
+<!-- 			    	<menuText>bbcontacts,bclcontact,casprr,ccmpred,comsat,epcmap,evfold,freecontact,gremlin,membrain,metapsicov,pconsc,pconsc2,pconsc3,pdb,plmdca,psicov</menuText>
+			    	<enumerators>bbcontacts,bclcontact,casprr,ccmpred,comsat,epcmap,evfold,freecontact,gremlin,membrain,metapsicov,pconsc,pconsc2,pconsc3,pdb,plmdca,psicov</enumerators> -->
+			    	<!--  Need default as setting it dynamically in the gui doesn't seem to work -->
+			    	<default>casprr</default>
```
Comment (fsimkovic) [inline i2/AMPLE.def.xml]: Could you please change this to `casp`? The code added in `ample/util/contact_util.py` disables the `casprr` option.

File: i2/AMPLE_gui.py [hunk 2]
```diff
@@ -148,6 +113,15 @@ def drawContents(self):
         self.closeSubFrame()
         
         self.drawOptions()
+        
+    def setContactFileOptions(self):
+        import conkit.io
+        parsers = sorted(conkit.io.CONTACT_FILE_PARSERS.keys())
```
Comment (fsimkovic) [inline i2/AMPLE_gui.py]: The ConKit mask is now in `ample/util/contact_util.py` and should be imported from there.
```
>>> from ample.util.contact_util import CONTACT_FILE_PARSERS
>>> print(CONTACT_FILE_PARSERS)
['bbcontacts', 'bclcontact', 'casp', 'ccmpred', 'comsat', 'epcmap', 
 'evfold', 'freecontact', 'gremlin', 'membrain', 'metapsicov', 'mmcif', 
 'pconsc', 'pconsc2', 'pconsc3', 'pdb', 'plmdca', 'psicov']
```
The default, as mentioned above, should be set to `casp`.

**Review by fsimkovic — CHANGES_REQUESTED** [review]

**Comment by linucks** [comment]
I've updated as per your comments.

**Comment by fsimkovic** [comment]
👍  for GUI in i2!!

**Review by fsimkovic — APPROVED** [review]

**Comment by fsimkovic** [comment]
Looks good to me. Unittests ran fine 👍 .

**Pull request merged**